### PR TITLE
Register REMOTE_DATA option

### PR DESCRIPTION
**Pull request by dstansby**
See https://github.com/matplotlib/matplotlib/issues/21668

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -49,6 +49,11 @@
 author = 'The SunPy Community'
 copyright = '{}, {}'.format(datetime.datetime.now().year, author)
 
+
+# Register remote data option with doctest
```
Comment (nabobalis) [inline docs/conf.py:53]: Does this need to go upstream to pytest-remote-data?

Comment (dstansby) [reply docs/conf.py:53]: If anywhere, to https://github.com/astropy/pytest-doctestplus - see discussion on https://github.com/matplotlib/matplotlib/issues/21668

Comment (nabobalis) [reply docs/conf.py:53]: Right that one!

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Some of those online fails are real but I guess we can deal with them in another PR.

**Comment by dstansby** [comment]
> Some of those online fails are real but I guess we can deal with them in another PR.

Here's one I made earlier: https://github.com/sunpy/sunpy/pull/5707

**Comment by dstansby** [comment]
Going to merge as this unblocks the doc build.

**Pull request merged**